src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String name, int radius, String color) {
+        super(name, color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String drawFigure() {
+        return "Figure: " + this.getName()
+                + ", area: " + this.getAria() + " sq.units, "
+                + "color: " + this.getColor();
```

Comment:
do we need `this` keyword here?

Reply:
I deleted
  + " sq.units, "
  And in the other class too.